apps/backend/src/user/user.service.ts

```diff
@@ -15,7 +17,7 @@ export class UserService {
    * TODO : passwords are stored clearly in database -> hash them !
    */
   async createUser(userDto: UserDto): Promise<number> {
-    const user = new UserEntity(userDto.username, userDto.password, userDto.role, userDto.active);
+    const user = new UserEntity(userDto.username, hashString(userDto.password), userDto.role, userDto.active);
```

Comment:
`UserEntity` class could use some kind of `fromDto` method instead.

Reply:
As discussed : to be done at next rewriting of User stuff